<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordRecoveryDialog</class>
 <widget class="QDialog" name="PasswordRecoveryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>找回密码</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource/icom.qrc">
    <normaloff>:/LoginORegister-Background.webp</normaloff>:/LoginORegister-Background.webp</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>52</y>
     <width>656</width>
     <height>420</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="phoneNumberLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>手机号:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phoneNumberLineEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="phoneNumberErrorlabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="captchaLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>验证码:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="captchaLineEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="showCaptchaLabel">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reflushButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resource/icom.qrc">
            <normaloff>:/refresh.png</normaloff>:/refresh.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="captchaErrorLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="passwordLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>密码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>50</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="passwordErrorLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="confirnPasswordLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>确认密码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="confirmPasswordLineEdit">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>50</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="confirmPasswordErrorLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="OkAndCancelButton">
      <item>
       <spacer name="horizontalSpacer_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_15">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Resource/icom.qrc"/>
 </resources>
 <connections/>
</ui>
